app/controllers/home_controller.rb

```diff
@@ -1,7 +1,13 @@
 class HomeController < ApplicationController
   before_action :set_user
 
-  def index; end
+  def index
+    @search = params[:search]
+    return @all_books = Book.all.order(:author) if @search.blank?
+
+    @author = @search[:author].strip
+    @all_books = Book.where('author collate "en_US" ILIKE ?', "%#{@author}%")
```

Comment:
What happens if instead of collating, you use the `lower` SQL function when performing the query? e.g. `Book.where('LOWER(author) LIKE ?', author.downcase)`

Reply:
This is solved now with creating new database with explicitly set locale to `ru_RU.UTF-8`